src/NServiceBus.Core.Tests/GlobalTestSetup.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.Core.Tests
+{
+    using System.Globalization;
+    using System.Threading;
+    using NUnit.Framework;
+
+    [SetUpFixture]
+    public class GlobalTestSetup
```

Comment:
@danielmarbach https://github.com/Particular/NServiceBus/pull/4068#discussion_r75151158

I suppose we could have renamed RedirectHelpers instead and centralized everything.